ui/DesignSystem/Component/Transaction/Summary.svelte

```diff
@@ -0,0 +1,46 @@
+<script>
+  import { Flex, Icon, Text } from "../../Primitive";
+
+  import * as transaction from "../../../src/transaction.ts";
+
+  export let summary = null;
+
+  const progress =
+    summary.progress /
+    (summary.counts.sum - summary.counts[transaction.StateType.Failed]);
+  const state = transaction.summaryIconState(summary.counts);
+  const text = transaction.summaryText(summary.counts);
+</script>
+
+<style>
+  .summary {
+    background-color: var(--color-background);
+    border-radius: 3px;
+    height: 56px;
+  }
+
+  .text {
+    display: flex;
+  }
+</style>
+
+<div class="summary" on:click>
+  <Flex>
+    <div slot="left" class="text">
+      <Icon.TransactionState
+        {progress}
+        required={transaction.REQUIRED}
```

Comment:
What is a 'required' transaction?

Reply:
It's the minimum amount of blocks that have to be mined on top of the one the tx is in for it to be settled.